README.md

```diff
@@ -1,9 +1,28 @@
 
 # AWS IoT Device SDK C 202009.00
 
+**[Link to API documentation]**    (LINK REQUIRED)
 
-## Branches
+We have released version 202009.00 of the AWS IoT SDK for Embedded-C (C-SDK). This release contains refactored libraries, and is a new design from the C-SDK v3.x. This release includes refactored MQTT and AWS IoT Device Shadow libraries for memory usage and modularity, and includes dependent libraries via GitHub submoduling. These libraries have gone through code quality checks including for [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html), [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx), [Coverity statical analysis](https://scan.coverity.com/), and [AWS CBMC automated reasoning tool](https://www.youtube.com/watch?v=YwQHAPRhQkI&feature=youtu.be&t=1721) to ensure memory safety, thread safety and functional correctness proof.   
```

Comment:
> "This release includes refactored MQTT and AWS IoT Device Shadow libraries for memory usage and modularity, and includes dependent libraries via GitHub submoduling."

This sentence can be unclear to the reader. Can it be something like: "These libraries include MQTT, JSON parser and AWS IoT Device Shadow libraries that have been refactored for modularity and memory usage. This repository includes these libraries through Git submoduling. "